integration-tests/src/test/java/com/datastax/oss/driver/api/core/tracker/RequestLoggerIT.java

```diff
@@ -132,9 +120,14 @@
               DefaultDriverOption.REQUEST_LOGGER_MAX_VALUES,
               RequestLogger.DEFAULT_REQUEST_LOGGER_MAX_VALUES)
           .withBoolean(DefaultDriverOption.REQUEST_LOGGER_STACK_TRACES, true)
-          .withProfile("low-threshold", lowThresholdProfile)
```

Comment:
I see that we cannot compose configs from other configs anymore. Not sure if this is worrisome?

Reply:
Yeah I lazily inlined everything, but I guess I could have instead created static methods that add the profiles to a given builder.

Reply:
I think it is useful to paste a profile. In users code, it may be often a case where profiles are created separately and finally, the list of profiles is set on the builder using `withProfile` for every entry.

Reply:
The same can be achieved with methods:
```java
ProgrammaticConfigBuilder addFooProfile(ProgrammaticConfigBuilder target) {
  return target
      .startProfile("foo")
      .withBoolean(DefaultDriverOption.REQUEST_LOGGER_STACK_TRACES, true)
      .endProfile();
}
```